packages/browser-agent-core/features/page-action/aggregate/index.js

```diff
@@ -26,13 +26,11 @@ export class Aggregate extends FeatureBase {
 
     if (document.referrer) this.referrerUrl = cleanURL(document.referrer)
 
-    this.ee.on('feat-ins', () => {
-      register('api-addPageAction', (...args) => this.addPageAction(...args), undefined, this.ee)
+    register('api-addPageAction', (...args) => this.addPageAction(...args), undefined, this.ee)
 
-      var scheduler = new HarvestScheduler('ins', {onFinished: (...args) => this.onHarvestFinished(...args)}, this)
-      scheduler.harvest.on('ins', (...args) => this.onHarvestStarted(...args))
-      scheduler.startTimer(this.harvestTimeSeconds, 0)
-    })
+    var scheduler = new HarvestScheduler('ins', {onFinished: (...args) => this.onHarvestFinished(...args)}, this)
```

Comment:
What's the implication with removing the 'feat-ins' event so that these lines execute on constructor? Do you also need to update `tests/browser/ins/add-page-action.browser.js` file that emits the 'feat-ins'?